kaspi_app/SwiftUI/ProductSwiftUI.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  ProductSwiftUI.swift
+//  kaspi_app
+//
+//  Created by Yersin Kazybekov on 22.12.2023.
+//
+
+import SwiftUI
+
+
+enum ProductCardType{
+    case large
+    case small
+}
+
+struct LastVisitedView: View{
+    var lastVisited = [
```

Comment:
for now keep it here, but in future we will move it to the Model/Presenter, because it is data, not UI